src/lib/constants.ts

```diff
@@ -68,8 +68,6 @@ export const KCL_DEFAULT_DEGREE = `360`
 /** localStorage key for the playwright test-specific app settings file */
 export const TEST_SETTINGS_FILE_KEY = 'playwright-test-settings'
 
-export const DEFAULT_HOST = 'https://api.zoo.dev'
-export const PROD_APP_URL = 'https://app.zoo.dev'
```

Comment:
Moved this one to an env variable controlled by vite on build, this way we can flip between dev and prod easier